facts/server.py

```diff
@@ -170,8 +170,19 @@ def process_request(request, tenantid, serverid):
         value = item['value']
 
         # Obtain the information of used memory and cpu
-        if name == 'usedMemPct' or name == 'cpuLoadPct' or name == 'netLoadPct' or name == 'freeSpacePct':
-            data.insert(len(data), float(value))
+        if name == 'usedMemPct':
```

Comment:
Because if this data comes in a different order, data will be inserted unordered.